app/src/main/res/values/strings.xml

```diff
@@ -50,8 +50,8 @@
     <string name="SettingsActivity.Title.Edit">Edit</string>
     <string name="SettingsActivity.Title.Save">Save</string>
     <string name="SettingsActivity.Title.EnterIntranetUrl">Enter your intranet URL</string>
-    <string name="SettingsActivity.Title.DeleteConfirmation">Delete server</string>
-    <string name="SettingsActivity.Message.DeleteConfirmation">Do you really want to delete this server ?</string>
+    <string name="SettingsActivity.Title.DeleteConfirmation">Delete intranet</string>
```

Comment:
Shouldn't we use eXo instead of intranet ? cc @hghorbel 
```suggestion
    <string name="SettingsActivity.Title.DeleteConfirmation">Delete eXo instance</string>
```

Reply:
@ahamdi I agree, as long as we do have as question: Do you really want to delete this eXo? so to make them consistent.
but should we have a FR version ?

Reply:
@BENABDESSALEM could we add an FR translation file ?

Reply:
@ahamdi Already exist

Reply:
@ahamdi @BENABDESSALEM for FR we will have: 
Supprimer instance eXo 
Voulez-vous vraiment supprimer cette instance eXo?

Reply:
@ahamdi @hghorbel Done